regen-network/regen-web
# feat: APP-195 files preview

## Description

https://regennetwork.atlassian.net/browse/APP-195?atlOrigin=eyJpIjoiYmIyYjMyYjRiMGEyNGJiNjllZjYyMGNkZGJiNDVmNDQiLCJwIjoiaiJ9

Also fixes https://regennetwork.atlassian.net/browse/APP-210



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

Check the files previews on:
https://deploy-preview-2403--regen-storybook.netlify.app/?path=/story/organisms-postfiles--public
https://deploy-preview-2403--regen-storybook.netlify.app/?path=/story/organisms-postfiles--private-locations

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
